web-app/src/app/tabs/race/race-graph/race-graph.component.ts

```diff
@@ -0,0 +1,205 @@
+import { Component, OnInit } from '@angular/core';
+import {MatRadioModule} from '@angular/material/radio';
+
+import { LapService } from '../../../_services/lap.service';
+import { LapData } from '../../../_objects/lapData';
+
+import * as CanvasJS from '../../../../assets/thirdParty/canvasjs.min.js';
```

Comment:
Why is this a * as opposed to { CanvasJS }

Reply:
Looks like that's how they did it in the example:
`https://canvasjs.com/angular-charts/`

Reply:
Yeah I was trying to import all the functionality in the src file. The file is not exactly legible